test/common/grpc/grpc_client_integration_test_harness.h

```diff
@@ -28,7 +29,8 @@
 #include "test/integration/fake_upstream.h"
 #include "test/mocks/grpc/mocks.h"
 #include "test/mocks/local_info/mocks.h"
-#include "test/mocks/server/mocks.h"
+#include "test/mocks/server/transport_socket_factory_context.h"
+
```

Comment:
minor: remove the extra newline here? (I'd expect the fix-formatting script to take care of this.)

Reply:
fix-formatting didn't remove newline like this. It has been removed manually.